api/v1/flow/flow.proto

```diff
@@ -477,17 +477,20 @@ message LostEvent {
 // AgentNotification in pkg/monitor/api/types.go
 enum AgentEventType {
     AGENT_EVENT_UNKNOWN = 0;
-    AGENT_STARTED = 1;
-    POLICY_UPDATED = 2;
-    POLICY_DELETED = 3;
-    ENDPOINT_REGENERATE_SUCCESS = 4;
-    ENDPOINT_REGENERATE_FAILURE = 5;
-    ENDPOINT_CREATED = 6;
-    ENDPOINT_DELETED = 7;
-    IPCACHE_UPSERTED = 8;
-    IPCACHE_DELETED = 9;
-    SERVICE_UPSERTED = 10;
-    SERVICE_DELETED = 11;
+    // used for AGENT_EVENT_GENERIC in monitor API, but there are currently no
+    // such events;
+    reserved 1;
```

Comment:
Generally speaking, we can't reassign values. I understand this breaking change is OK here as `AgentEventType` is not in any stable Cilium release.

Reply:
Indeed, I should have mentioned this is a breaking change in the commit message at least. As you pointed out it is not part of any stable release, so I figure it should be fine.